tests/test_util.fish

```diff
@@ -37,19 +37,21 @@ if not set -q __fish_is_running_tests
     end
     set -l IFS # clear IFS so cmd substitution doesn't split
     cd (dirname $script); or die
-    set -xl XDG_CONFIG_HOME (printf '%s/tmp.config/%s' $PWD %self); or die
-    if test -d $XDG_CONFIG_HOME
-        # if the dir already exists, we've reused a pid
-        # this would be surprising to reuse a fish pid that failed in the past,
-        # but clear it anyway
-        rm -r $XDG_CONFIG_HOME; or die
-    end
+
+    set -lx XDG_DATA_HOME ../test_data
```

Comment:
I don't remember anymore why I used the PID. I suspect I wanted to be prepared for the ability to run `make test` multiple times simultaneously (tests that create /tmp files with invariant names really should be fixed) but it's been long enough that I'm basically just guessing.